docker-compose.yml

```diff
@@ -5,17 +5,34 @@ services:
     restart: always
     volumes:
       - lodestar:/root/.local/share/lodestar
-    command: beacon --network ${NETWORK:-pyrmont}
+      - /root/.local/share/lodestar/pyrmont:/root/.local/share/lodestar/pyrmont
```

Comment:
Why are you mapping the volume twice, and only for the pyrmont network?

Reply:
I want to map the manually imported validators for pyrmont.  Do you have a suggestion for doing it a different way, or is this okay?

Reply:
You should use the CLI flags to use a constant directory for the keystores independent of the network. This setup must work for mainnet and pyrmont